user_deletion/managers.py

```diff
@@ -0,0 +1,22 @@
+from dateutil.relativedelta import relativedelta
+from django.apps import apps
+from django.utils import timezone
+
+
+user_deletion_config = apps.get_app_config('user_deletion')
+
+
+class UserManagerMixin:
+    def users_to_notify(self):
+        """Finds all users who have been inactive and not yet notified."""
+        last_login = timezone.now() - relativedelta(
+            months=user_deletion_config.MONTH_NOTIFICATION,
+        )
```

Comment:
updated in https://github.com/incuna/django-user-deletion/pull/2/commits/7ce51c694e44e8503acd86de0f90dbc4078f4b82